isis/src/base/objs/PvlKeyword/PvlKeyword.cpp

```diff
@@ -1528,6 +1528,15 @@ namespace Isis {
       }
     }
 
+    QRegularExpression regex("(,)");
+    QString temp = "";
+    temp += keyword[0];
+    QRegularExpressionMatch match = regex.match(temp);
```

Comment:
What is the regex trying to do here? If you are looking for an arbitrary list of leftover band numbers, something like `(,*[0-9]+)+` would probably be better. This looks more like it's just checking for a comma which is basically the same as `keyword[0] == ','`